py/selenium/webdriver/remote/webelement.py

```diff
@@ -105,7 +106,7 @@ def get_attribute(self, name):
             attributeValue = None
         else:
             attributeValue = resp['value']
-            if name != 'value' and attributeValue.lower() in ('true', 'false'):
+            if name != 'value' and type(attributeValue) is StringType and attributeValue.lower() in ('true', 'false'):
```

Comment:
what about unicode?  will that work here?

also, I think it is preferred to check `isinstance` rather than using `type`:

`if isinstance(item, str):`

this would also handle unicode, since `str` is aliased to `basestring` in this module.